### PR TITLE
#441 Upgrade Artemis image to 2.2.0-4

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

**Comment by dejanb** [comment]
Updated the PR to 2.2.0-4 image version

File: bom/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <description>A &quot;bill-of-materials&quot; for Hono.</description>
 
   <properties>
-    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-3</artemis.image.name>
+    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-4</artemis.image.name>
```
Comment (sophokles73) [inline bom/pom.xml]: my understanding is that this one does no longer include JGroups, right?

Comment (dejanb) [reply bom/pom.xml]: Yes, and it has been already approved https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15347

**Comment by dejanb** [comment]
Is it possible to leave JGroups CQ open, so we can finish the investigation and decide further steps without blocking Hono release?

**Pull request merged**